lib/modules/apostrophe-images/index.js

```diff
@@ -80,6 +80,50 @@ module.exports = {
       }
     ].concat(options.arrangeFields || []);
 
+    options.addFilters = [
+      {
+        name: 'orientation',
+        choices: [
+          {
+            value: 'landscape',
+            label: 'Lansdscape'
+          },
+          {
+            value: 'portrait',
+            label: 'Portrait'
+          },
+          {
+            value: 'square',
+            label: 'Square'
+          }
+        ],
+        def: ''
+      },
+      {
+        name: 'fileType',
+        choices: [
+          {
+            value: 'jpg',
```

Comment:
The list of extensions is dynamic, it depends on `self.apos.attachments.fileGroups`, which is an array. One of them will have `name: 'images'`, use the `extensions` property of that to figure out what is supported. Note that svg is not turned on for every site.

See apostrophe-attachments/index.js

Reply:
Thanks for this info.